addons/mrp/models/mrp_unbuild.py

```diff
@@ -144,6 +140,7 @@ def action_unbuild(self):
         if self.product_id.tracking != 'none' and not self.lot_id.id:
             raise UserError(_('You should provide a lot number for the final product.'))
 
+        # TODO: remove me in master
```

Comment:
Don't leave this comment please. you will have the possibility to remove it in the last forward bot (which target the master).